addons/my-ocular/addon.json

```diff
@@ -1,15 +1,41 @@
 {
-  "name": "Show my-ocular status",
-  "description": "Shows my-ocular statuses under usernames on the forums. Uses my-ocular.jeffalo.net to obtain status data. Statuses are moderated and filtered.",
+  "name": "Show ocular status",
+  "description": "Shows ocular statuses and favorite colours. Uses my-ocular.jeffalo.net to obtain status data. Statuses are moderated and filtered.",
   "credits": [
     {
       "name": "Jeffalo"
     }
   ],
   "userscripts": [
     {
-      "url": "userscript.js",
-      "matches": ["https://scratch.mit.edu/discuss/topic/*"]
+      "url": "discuss.js",
+      "matches": ["https://scratch.mit.edu/discuss/topic/*"],
+      "settingMatch": {
+        "id": "discuss",
+        "value": true
+      }
+    },
+    {
+      "url": "profile.js",
+      "matches": ["https://scratch.mit.edu/users/*"],
```

Comment:
Might want to add a "/" at the end... otherwise you can go to https://scratch.mit.edu/users/Scratchteam/following/ for example and you will get an error in the console.

Reply:
fix